packages/create-turbo/__tests__/cli.test.ts

```diff
@@ -77,7 +77,11 @@ describe("create-turbo cli", () => {
         case 3:
           // Which package manager do you want to use?
           // easy to change deployment targets.
-          expect(getPromptChoices(prompt)).toEqual(["npm", "pnpm", "yarn"]);
+          expect(getPromptChoices(prompt)).toEqual([
+            "npm",
+            "pnpm",
+            "- yarn (not installed)",
```

Comment:
this is not great, but that's what it is right now in CI (and also locally for me). i could add yarn setup to the action steps to make this better, but might be better as a follow up PR

Reply:
Does this fix itself with `corepack` being enabled globally?

Reply:
I don't think that will affect this. It won't install yarn for us unless we tell it to. This output just reflects the state of the system when the CLI runs:

https://github.com/vercel/turborepo/blob/655f73ed152943e26aee6cef0fa17f11984ccc01/packages/create-turbo/src/index.ts#L118-L129

Reply:
Right, but it determines what versions are available by running `yarnpkg -v` which is a shim that will exist of `corepack` is enabled.